test/get/hosted/advisory_test.dart

```diff
@@ -190,6 +190,36 @@ Future<void> main() async {
     await ctx.run(['get']);
   });
 
+  testWithGolden('more than one affected package with pub as ecosystem',
```

Comment:
```suggestion
  testWithGolden('more than one affected package-version with pub as ecosystem',
```
Or would it be better to actually affect two different packages?